adbpyg_adapter/utils.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+from typing import Any, Dict, Optional
+
+from pandas import DataFrame
+from rich.progress import Progress, SpinnerColumn, TextColumn, TimeElapsedColumn
+from torch import Tensor, from_numpy, zeros
+
+logger = logging.getLogger(__package__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    f"[%(asctime)s] [{os.getpid()}] [%(levelname)s] - %(name)s: %(message)s",
+    "%Y/%m/%d %H:%M:%S %z",
+)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def progress(
+    text: str,
+    text_style: str = "none",
+    spinner_name: str = "aesthetic",
+    spinner_style: str = "#5BC0DE",
+    transient: bool = False,
+) -> Progress:
+    return Progress(
+        TextColumn(text, style=text_style),
+        SpinnerColumn(spinner_name, spinner_style),
+        TimeElapsedColumn(),
+        transient=transient,
+    )
+
+
+class EnumEncoder(object):
```

Comment:
Could be worth splitting these out into their own `encoders.py` ?

Reply:
ya let's do that

Reply:
addressed in [ab082a5](https://github.com/arangoml/pyg-adapter/pull/2/commits/ab082a5921e78b1e9e6e0c679bb9f1f6210bf418)